lib/airbrake-ruby/sync_sender.rb

```diff
@@ -33,18 +21,15 @@ def initialize(config)
     # @param [Airbrake::Notice] endpoint
     # @return [Hash{String=>String}] the parsed HTTP response
     def send(notice, endpoint = @config.endpoint)
-      response = nil
       req = build_post_request(endpoint, notice)
       https = build_https(endpoint)
 
-      begin
-        response = https.request(req)
-      rescue *HTTP_ERRORS => ex
-        @config.logger.error("#{LOG_LABEL} HTTP error: #{ex}")
-        return
-      end
-
+      response = https.request(req)
       Response.parse(response, @config.logger)
+
+    rescue => ex
```

Comment:
But if `build_post_request` or `build_https` raise and exception and we don't capture it, the async worker is going to die, which is what we are trying to prevent.